server/controllers/inventory/depots/index.js

```diff
@@ -110,12 +111,27 @@ function update(req, res, next) {
 * @function list
 */
 function list(req, res, next) {
-  var sql =
-    `SELECT BUID(uuid) as uuid, text, is_warehouse
+  const options = req.query;
+  options.enterprise_id = req.session.enterprise.id;
+
+  const filters = new FilterParser(options, { tableAlias : 'depot' });
+
+  var sql =`
+    SELECT BUID(uuid) as uuid, text, is_warehouse
     FROM depot
-    WHERE enterprise_id = ?;`;
+  `;
+
+  filters.custom(
```

Comment:
That's interesting!  I never thought about adding this as a feature on the depots controller.